library/src/main/java/com/novoda/downloadmanager/LiteDownloadManagerDownloader.java

```diff
@@ -81,12 +85,9 @@ void download(DownloadBatch downloadBatch, Map<DownloadBatchId, DownloadBatch> d
 
         DownloadBatch batchToDownload = downloadBatchMap.get(downloadBatchId);
 
-        executor.submit(new Runnable() {
-            @Override
-            public void run() {
-                Wait.<Void>waitFor(serviceCriteria, waitForDownloadService)
-                        .thenPerform(executeDownload(batchToDownload, downloadBatchMap));
-            }
+        executor.submit(() -> {
+            Wait.<Void>waitFor(serviceCriteria, waitForDownloadService)
```

Comment:
Did this convert it to a `Callable` or a `Runnable`? I believe this is why it wasn't inlined before.

Reply:
My IDE tells me that it is still runnable as it was before, however I will revert this change, because it is not part of this PR